econml/bootstrap.py

```diff
@@ -5,6 +5,254 @@
 import numpy as np
 from joblib import Parallel, delayed
 from sklearn.base import clone
+from scipy.stats import norm
+from collections import OrderedDict
+import pandas as pd
+
+
+class BootstrapInferenceResults:
```

Comment:
Why isn't this a child of the InferenceResults and re-use a lot of the code there?

Reply:
In fact InferenceResults already contain an input param pred_dist in anticipation of filling in the bootstrap. Not sure what the best merging is, but the two classes need to be merged somehow or be under the same inheritance family to re-use a lot of the logic.

Reply:
The hierarchy is now much improved and shares most logic.